keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -327,9 +356,9 @@ No changes needed
 ###### How can this feature be enabled / disabled in a live cluster?
 
 - [x] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: `TopologyManagerPolicyOptions`
+  - Feature gate name: `TopologyManagerPolicyAlphaOptions`
   - Components depending on the feature gate: kubelet
-- [x] Change the kubelet configuration to set a `TopologyManager` policy to `restricted` or `best-effort` and a `TopologyManagerPolicyOptions` to `prefer-closest-numa-nodes`
+- [x] Change the kubelet configuration to set a `TopologyManager` policy to `restricted` or `best-effort` and a `TopologyManagerPolicyAlphaOptions` to `prefer-closest-numa-nodes`
```

Comment:
This should be called TopologyManagerPolicyOptions.

Reply:
Fixed.